.gitignore

```diff
@@ -24,6 +24,9 @@ ios/
 *.ipr
 .idea/
 
+# VSCode
+.vscode/
+
```

Comment:
???

Reply:
Tried dart in vscode today and yeah thils folder was missing in the gitignore